cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
```

Comment:
Do we need this file? It doesn't seem to add anything and failing the linter.

Reply:
For what I read is for the intellisense